doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -78,145 +78,109 @@ Displaying
 Flags, Options and Tables
 -----------------------------
 
-|Coq| configurability is based on flags (e.g. :flag:`Printing All`), options
-(e.g. :opt:`Printing Width`), or tables (e.g. :cmd:`Add Printing Record`). The
-names of flags, options and tables are made of non-empty sequences of
-identifiers (conventionally with capital initial letter). The general commands
-handling flags, options and tables are given below.
+Coq has many settings to control its behavior.  Setting types include flags, options
+and tables.  Flags are boolean values, such as :flag:`Printing All`.  Options have numeric or
+string values, such as :opt:`Printing Width`.  Tables contain sets of strings.
+(In addition, some commands just provide settings, for example :cmd:`Extraction Language OCaml`.)
 
-.. TODO : flag is not a syntax entry
-
-.. cmd:: Set @flag
-
-   This command switches :n:`@flag` on. The original state of :n:`@flag`
-   is restored when the current module ends.
-
-   .. cmdv:: Local Set @flag
-
-      This command switches :n:`@flag` on. The original state
-      of :n:`@flag` is restored when the current *section* ends.
-
-   .. cmdv:: Global Set @flag
-
-      This command switches :n:`@flag` on. The original state
-      of :n:`@flag` is *not* restored at the end of the module. Additionally, if
-      set in a file, :n:`@flag` is switched on when the file is `Require`-d.
-
-   .. cmdv:: Export Set @flag
-
-      This command switches :n:`@flag` on. The original state
-      of :n:`@flag` is restored at the end of the current module, but :n:`@flag`
-      is switched on when this module is imported.
+Flags, options and tables are identified by a series of identifiers, each with an initial
+capital letter.
 
+.. TODO : flag is not a syntax entry
 
-.. cmd:: Unset @flag
-
-   This command switches :n:`@flag` off. The original state of
-   :n:`@flag` is restored when the current module ends.
-
-   .. cmdv:: Local Unset @flag
-
-      This command switches :n:`@flag` off. The original
-      state of :n:`@flag` is restored when the current *section* ends.
-
-   .. cmdv:: Global Unset @flag
-
-      This command switches :n:`@flag` off. The original
-      state of :n:`@flag` is *not* restored at the end of the module. Additionally,
-      if set in a file, :n:`@flag` is switched off when the file is `Require`-d.
+.. cmd::  {? Local | Global | Export } Set @flag
+   :name: Set
 
-   .. cmdv:: Export Unset @flag
+   Sets :n:`@flag` on. Scoping qualifiers are
```

Comment:
Both forms are OK.  "Turns the flag on" is a little more idiomatic, but since the commands are "Set" and "Unset", I thought it would be better for readers to see the same verbs in the description.